src/sync/SyncService.ts

```diff
@@ -1,20 +1,23 @@
-import {StateType} from "./StateType";
-import {Observe} from "../Observer";
+import {StateType} from "../StateType";
 import {OfflineDataService} from "./OfflineDataService";
+import {ServiceStatus} from "../ServiceStatus";
+import {defaultPingService} from "../PingService";
 
 export enum SyncStatus {
     WAITING,
     NO_DATA,
     DATA
 }
 
-@Observe()
+var sss = new ServiceStatus(defaultPingService);
+sss.Observe();
```

Comment:
why @ is omitted from sss.observe()
And also can we give a proper name instead of "sss"